src/scheduler/JobScheduler.ts

```diff
@@ -67,6 +68,7 @@ export class JobScheduler {
   }
 
   async start(): Promise<void> {
+    // Make sure the job is actually stopped. This should only be necessary when mongo errors prevented momo from correctly updating its state
```

Comment:
Not sure about this comment.

(Not sure about how restarting jobs actually works currently, tbh)

Reply:
Maybe we have a bug that prevents you from stopping and restarting a schedule, but we should probably look into that not in this PR ...

Reply:
Yeah, agreed. Can we formulate an issue for this? Not sure how else to track it.

Reply:
Will try to do that later